virttest/utils_test/__init__.py

```diff
@@ -447,70 +447,62 @@ def run_file_transfer(test, params, env):
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     login_timeout = int(params.get("login_timeout", 360))
-
     error_context.context("Login to guest", logging.info)
-    session = vm.wait_for_login(timeout=login_timeout)
```

Comment:
Sounds reasonable, we need to ensure VM really bootup before try to copy file to/from it. thanks your comments.

Thanks,
Xu